financial-connections/src/main/java/com/stripe/android/financialconnections/model/FinancialConnectionsAuthorizationSession.kt

```diff
@@ -26,7 +26,7 @@ internal data class FinancialConnectionsAuthorizationSession(
     val id: String,
 
     @SerialName(value = "next_pane")
-    val nextPane: FinancialConnectionsSessionManifest.NextPane,
+    val nextPane: FinancialConnectionsSessionManifest.ClientPane,
```

Comment:
on iOS we just added extra cases to the `NextPane` enum

and then added a comments on which of these are client-only

Reply:
`ClientName` is not a good name, realizing from your comment that is confusing. Renamed just to `Pane` (this includes both client and backend panes). https://github.com/stripe/stripe-android/pull/5881/commits/4470bf1adebe566992f1625068f9d67b03897fc0